MuonAnalysis/MomentumScaleCalibration/plugins/DBWriter.cc

```diff
@@ -64,10 +64,9 @@ void DBWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
   edm::Service<cond::service::PoolDBOutputService> mydbservice;
   if (mydbservice.isAvailable()) {
     if (mydbservice->isNewTagRequest("MuScleFitDBobjectRcd")) {
-      mydbservice->createNewIOV<MuScleFitDBobject>(
-          dbObject, mydbservice->beginOfTime(), mydbservice->endOfTime(), "MuScleFitDBobjectRcd");
+      mydbservice->createOneIOV<MuScleFitDBobject>(dbObject, mydbservice->beginOfTime(), "MuScleFitDBobjectRcd");
```

Comment:
Is it worth considering to merge the two calls into writeOneIov?

Reply:
I had the same doubt but the original code created a new IOV at `beginOfTime()` and appends a new one at `currentTime()`, and from the lines in: https://github.com/cms-sw/cmssw/blob/1676d5e26e2c3a1d5868fd5b7ceaa755b52364dc/CondCore/DBOutputService/interface/PoolDBOutputService.h#L89-L93 I dont' think `writeOneIOV` is properly taking this into account. Or am I missing something?

Reply:
Well I guess the idea is that for a new tag, is preferred an IOV with since=1. Otherwise, the IOV sequence would not support target time < currentTime... But this has never been taken into account in all the cases treated previously?

Reply:
As discussed privately with @ggovi the best solution would probably be to handle everything inside the `writeOneIOV` method definition in `CondCore`. 
But for now I think this is the best way to go, and we should evaluate case by case when we do this `createNewIOV+appendSince --> writeOneIOV` migration (like I did in this case).
@ggovi do you agree?

Reply:
@francescobrivio I agree, we need case by case evaluation...